tests/test_templatetags_component.py

```diff
@@ -12,22 +12,28 @@
 setup_test_config({"autodiscover": False})
 
 
-class SlottedComponent(Component):
-    template_file = "slotted_template.html"
+def gen_slotted_component():
```

Comment:
I wrapped the components in functions so that the component creation is done within a function that is wrapped in `@djc_test` decorator to fix some global state issue with tests.